doc/data/commands/ttn-lw-cli.json

```diff
@@ -3,6 +3,62 @@
     "short": "The Things Industries Command-line Interface",
     "path": "ttn-lw-cli",
     "subCommands": {
+      "alert-notification-profiles": {
+        "short": "Alert Notification Profile commands (EXPERIMENTAL)",
+        "path": "ttn-lw-cli alert-notification-profiles",
```

Comment:
```suggestion
        "path": "ttn-lw-cli alert_notification_profiles",
```

Reply:
I changed path to this and local build was successful, can you try that?

Reply:
This was also my mistake, in the path there were two distinct files: one with `alert-notification-profiles` (generated by the commands you gave and another `alert_notifications_profiles` which I created. 

Everything is working now, thanks for the help